packages/dev/core/src/Rendering/iblCdfGenerator.ts

```diff
@@ -162,6 +166,10 @@ export class IblCdfGenerator {
         if (this._iblSource!.isCube) {
             size.width *= 4;
             size.height *= 2;
+            // Force the resolution to be a power of 2 because we rely on the
+            // auto-mipmap generation for the scaled luminance texture.

```

Comment:
I'm not sure I understand this comment, we can generate mipmaps for textures that are not powers of 2?